python/src/aac/cli.py

```diff
@@ -65,7 +66,7 @@ def _validate_cmd(model_file: str) -> PluginExecutionResult:
     """Run the built-in validate command."""
 
     def validate_model() -> str:
-        with validation(parser.parse, model_file):
+        with validate_source(model_file):
```

Comment:
I thought that I had already commented this, but it looks like the pattern is -ed if the context manager performs the action on enter or -ing if it performs the action on exit. Since the validation context executes the validation before entering the context scope, then we will want the -ed version. Thanks!
https://localcoder.org/naming-convention-for-context-manager-classes-with-blocks